pkg/scalers/kafka_scaler.go

```diff
@@ -121,9 +121,23 @@ func NewKafkaScaler(config *ScalerConfig) (Scaler, error) {
 
 func parseKafkaAuthParams(config *ScalerConfig, meta *kafkaMetadata) error {
 	meta.saslType = KafkaSASLTypeNone
+	var saslAuthType string
+	switch {
+	case config.TriggerMetadata["saslAuthType"] != "":
+		saslAuthType = config.TriggerMetadata["saslAuthType"]
```

Comment:
Good catch! Agree, we should use the same key for both TA and trigger metadata.

Reply:
Agree with that as I cross commented on the doc PR. Also it means that for tls the value should be enable/disable instead of true/false right? Just want to confirm this

Reply:
yeah, the valid values shouldn't change from previous state

Reply:
Got it. Thanks for the review. I will try do it later once im back to keyboard